code/src/components/Artists/Artists.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { SvgIcons } from '../SvgIcons/SvgIcons';
+import data from '../../data.json';
+import './Artists.css';
+
+export const Artists = () => {
+  return (
+    <div className="album-wrapper">
+      {data.albums.items.map((album) => {
+        return (
+          <div className="single-album-container" key={album.id}>
+            <div className="album-cover-div">
+              <img
+                className="album-cover"
+                alt="Album cover"
+                src={album.images[0].url}
+                // eslint-disable-next-line react/jsx-closing-bracket-location
+              />
+              <SvgIcons />
+            </div>
+            <span className="album-name">{album.name}</span>
+            <div className="artist-names-div">
+              {album.artists.map((artist, index) => (
+                <span className="artist-name" key={artist.id}>
+                  {' '}
+                  <a
+                    href={`https://open.spotify.com/artist/${artist.id}`}
```

Comment:
This was smart! We did it in another way: <Albums
              hrefAlbum={singleAlbumItem.external_urls.spotify}
              album={singleAlbumItem.name} />